src/EPR.SubsidiaryBulkUpload.Application/DTOs/CompaniesHouseCompanyMap.cs

```diff
@@ -42,13 +42,22 @@ private static List<UploadFileErrorModel> GetRowValidationErrors(IReaderRow row)
                     lineNumber, rawData, BulkUpdateErrors.OrganisationNameRequiredMessage, BulkUpdateErrors.OrganisationNameRequired));
         }
 
-        if (string.IsNullOrEmpty(row.GetField(nameof(CompaniesHouseCompany.companies_house_number))) && string.IsNullOrEmpty(row.GetField(nameof(CompaniesHouseCompany.franchisee_licensee_tenant))))
+        if (string.IsNullOrEmpty(row.GetField(nameof(CompaniesHouseCompany.companies_house_number)))
+            && string.IsNullOrEmpty(row.GetField(nameof(CompaniesHouseCompany.franchisee_licensee_tenant))))
         {
             errors.Add(
                 CreateError(
                     lineNumber, rawData, BulkUpdateErrors.CompaniesHouseNumberRequiredMessage, BulkUpdateErrors.CompaniesHouseNumberRequired));
         }
 
+        if (!string.IsNullOrEmpty(row.GetField(nameof(CompaniesHouseCompany.companies_house_number)))
```

Comment:
There should also be a check for spaces in the companies house number, with a different error message

Reply:
done